guidelines-for-respectful-communication/index.html

```diff
@@ -38,7 +38,7 @@ <h2>Guidelines For Respectful Communication</h2>
       <li>Where we disagree with someone, we avoid forms of expression that might make our dialogue partner feel attacked, humiliated, demeaned, or marginalised. Our critique should always be of specific statements and claims, never of people.</li>
       <li>Disagreement itself is fine: we are enriched by robust technical debate. But we seek to make the tone of that debate to be a conversation among people who respect, or even admire, each other.</li>
       <li>Where we disagree, we try to be curious about the perspective, goals, motivation, and priorities of the other person.</li>
-      <li>We do not tolerate any form of discriminatory language or behaviour towards any minority (for example age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation).</li>
+      <li>We welcome and support people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, colour, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability. </li>
```

Comment:
Suggestions
* Make this the first bullet.
* Stick to "We welcome as members of Haskell community people of all backgrounds...".  The "and support" is true but it's an extra distraction (what does it mean exactly?)
* Make the intro sentence "Specifically, we aspire to these behaviours:", rather than "qualities".  Behaviour is what you do; quality is more what you are.
* Consider following the bullets with "All these guidelines apply to our *behaviour*; that is, what we do.  They specifically do not, and should not, seek to constrain anyone's beliefs; that is, what they think."

Reply:
I find all your suggestions excellent. Please double check I got it right.